csharp/extractor/Semmle.Extraction.CSharp/Populators/AstLineCounter.cs

```diff
@@ -0,0 +1,48 @@
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Semmle.Util;
+
+namespace Semmle.Extraction.CSharp.Populators
+{
+    internal class AstLineCounter : CSharpSyntaxVisitor<LineCounts>
```

Comment:
Now `Method` no longer needs to be `partial`.

Reply:
Ahh, good point. Sorry...